netket/nn/linear.py

```diff
@@ -314,6 +318,95 @@ def __call__(self, inputs: Array) -> Array:
         return y
 
 
+class DenseEquivariant(Module):
+    """An equivariant layer that takes a feature map with embeddings corresponding to various
+    poses of a symmetry group, shape = [n_symm*in_features] and outputs a feature map of identical
+    shape [n_symm*out_features]. This transformation respects the group structure such that
+    g L({f}) = L(g{f}) where g is an element of the symmetry group and {f} are the feature embeddings.
+    """
+
+    group_algebra: HashableArray
+    """Flattened array generated  SymmGroup.group_algebra() that specifies the mapping
+    between absolute and relative poses """
+    in_features: int
+    """The number of symmetry-reduced input features. The full input size
```

Comment:
I'm not sure that this is correct. The input/output size will also depend on batches... I think giving an explicit example with what the shapes should be would be clearer.